code/Game.cs

```diff
@@ -50,38 +58,36 @@ private void ChangeRound(Round round)
 
                 case Game.Round.InProgress:
                     TimeRemaining = TTTRoundTime;
+                    
+                    int detectiveCount = (int) (All.Count * 0.125f);
+                    int traitorCount = (int) Math.Max(All.Count * 0.25f, 1f);
 
-                    #region Select Roles
-                    int detectiveCount = (int) ((float) Sandbox.Player.All.Count * 0.125f);
-                    int traitorCount = (int) Math.Max((float) Sandbox.Player.All.Count * 0.25f, 1f);
-
-                    List<Player> _players = Sandbox.Player.ConvertAll(p => (Player) p);
+                    var players = Client.All.ToList().ConvertAll(p => p.Pawn as TTTPlayer);
                     Random random = new Random();
 
                     // SELECT DETECTIVES
-                    for (int i = 0; i < detectiveCount; i++)
+                    for (var i = 0; i < detectiveCount; i++)
                     {
-                        int randomID = random.Next(_players.Count);
-                        _players[randomID].PlayerRole = Player.Role.Detective;
+                        var randomId = random.Next(players.Count);
+                        players[randomId].CurrentRole = Role.Detective;
```

Comment:
```
Yea you renamed it to avoid issues with the Role enum (something I documented in the Trello as well), but I think Role would be more intuitive. Instead, we probably should move or rename the Role enum.
```

Not sure if I understand exactly, but...

I've moved the `RoleType` enum inside of the `Player` class.